liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -202,13 +209,22 @@ protected Plugin getPlugin(String groupId, String artifactId) {
     }
     
     protected Plugin getLibertyPlugin() {
+        // Try getting the version from Maven 3's plugin descriptor
+        String version = null;
+        if (plugin != null && plugin.getPlugin() != null) {
+            version = plugin.getVersion();
+            log.debug("Setting plugin version to " + version);
+        }
         Plugin plugin = project.getPlugin(LIBERTY_MAVEN_PLUGIN_GROUP_ID + ":" + LIBERTY_MAVEN_PLUGIN_ARTIFACT_ID);
```

Comment:
simpler to read if  we used a different name for the local var than the field name

Reply:
Updated